deploy/kiali/kiali_cr.yaml

```diff
@@ -408,15 +408,26 @@ spec:
 # In this case, you would set "external_services.grafana.auth.password" to "secret:myGrafanaCredentials:myGrafanaPw".
 #
 # **Custom-dashboards settings:
-# enabled: enable or disable custom dashboards, including the dashboards discovery process. Default: true.
+# discovery_enabled: Enable, disable or set 'auto' mode to the dashboards discovery process. If set to true, Kiali
+#   will always try to discover dashboards based on metrics. Note that it can generate performance penalties while
+#   discovering dashboards for workloads having many pods (thus many metrics), especially with versions of Prometheus < 2.24.
+#   When set to 'auto', Kiali will skip dashboards discovery for workloads with more than a configured threshold of pods
+#   (see 'discovery_auto_threshold'). When discovery is disabled or auto/skipped, it is still possible to tie workloads
+#   with dashboards through annotations on pods (refer to the doc https://kiali.io/documentation/latest/runtimes-monitoring/#pods-annotations)
+#   Allowed values: true, false, auto. Default: auto.
+# discovery_auto_threshold: Threshold of pods, for a given Application or Workload, above which dashboards discovery will be skipped
+#   when in 'auto' mode. Default: 10.
+# enabled: Enable or disable custom dashboards, including the dashboards discovery process. Default: true.
```

Comment:
All peer settings in the operator config files are sorted alphabetically. Sort these alphabetically. Also, tweaked some of this for more clarity, and no need to mention the "Default" in the text of the description - all the commented values in this example CR file are the documented default values.

```suggestion
# discovery_auto_threshold: Threshold of the number of pods, for a given Application or Workload, above which dashboards discovery will be skipped.
#   This setting only takes effect when discovery_enabled is set to 'auto' mode.
# discovery_enabled: Enable, disable or set 'auto' mode to the dashboards discovery process. If set to true, Kiali
#   will always try to discover dashboards based on metrics. Note that it can generate performance penalties while
#   discovering dashboards for workloads having many pods (thus many metrics), especially with versions of Prometheus < 2.24.
#   When set to 'auto', Kiali will skip dashboards discovery for workloads with more than a configured threshold of pods
#   (see 'discovery_auto_threshold'). When discovery is disabled or auto/skipped, it is still possible to tie workloads
#   with dashboards through annotations on pods (refer to the doc https://kiali.io/documentation/latest/runtimes-monitoring/#pods-annotations)
#   Allowed values: true, false, auto.
# enabled: Enable or disable custom dashboards, including the dashboards discovery process.
```